gradlew

```diff
@@ -110,10 +125,11 @@ if $darwin; then
     GRADLE_OPTS="$GRADLE_OPTS \"-Xdock:name=$APP_NAME\" \"-Xdock:icon=$APP_HOME/media/gradle.icns\""
 fi
 
-# For Cygwin, switch paths to Windows format before running java
-if $cygwin ; then
+# For Cygwin or MSYS, switch paths to Windows format before running java
+if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
```

Comment:
About all the changes of this file, I am just curious (and not familiar with such scripts): Have you overtaken this from any more modern installation file from Android?

Reply:
As I said in [this comment](https://github.com/MenoData/Time4A/pull/2#discussion_r365571866), changes in these files (gradlew, gradlew.bat, gradle-wrapper.jar, and gradle-wrapper.properties) has been applied by running `gradlew wrapper` command, it's a command for upgrading project's Gradle wrapper  and it's the script files